### ISSUE TITLE
Detects a regular system as a VM

### ISSUE THREAD

**Issue by Sly-Little-Fox** [sebhildebrandt/systeminformation#532]
**Describe the bug**

**To Reproduce**
1. Start `systeminformation.system()`.
2. The output says that my computer is virtual, but it's not.
**Current Output**
```js
 virtual: true,
 virtualHost: 'VMware'
 ```
**Expected behavior**
It should write that the system is not a virtual machine.

**Environment (please complete the following information):**
 - systeminformation package version: 5.6.19
 - OS: Linux
 - Hardware: Some computer

**Additional context**
Nothing

**Comment by Sly-Little-Fox** [comment]
> @Sly-Little-Fox : Some questions to be able to identify the problem:
> 
>     1. Which linux distro you are running?
> 
>     2. Any details about your hardware, you are able to share?
> 
>     3. Can you run the following commands and share the results:
> 
> 
> ```
> cat /sys/devices/virtual/dmi/id/product_name
> cat /sys/devices/virtual/dmi/id/sys_vendor
> ls -1 /dev/disk/by-id/
> dmidecode -t 4
> ```
> 
> Thank you for your support

1. I use Linux Xubuntu 20.04.
2. Self-built computer with B450 AORUS PRO. 
3. 
```
$ cat /sys/devices/virtual/dmi/id/product_name
B450 AORUS PRO

$ cat /sys/devices/virtual/dmi/id/sys_vendor
Gigabyte Technology Co., Ltd.
```

I can't run dmidecode as I am not root . Sorry, I am afraid that there can be some confidential information in `ls -1 /dev/disk/by-id/` 🤦🏻‍♂️

**Comment by sebhildebrandt** [comment]
@Sly-Little-Fox Thank you for sharing. I fully understand that you do not want to share confidential information.

What would be interesting, what part of my code then determines that this is a virtual machine ... this happens in `lib/system.js` lines 80 to 180 ... would you be able to debug this and make a little investigation WHERE exactly `system.virtual` is set to true there (and virtualHost to VMWare) ... thank you in advance. 

Based on your output:
- lines 81-98 should not set virtual to true ... model is `B450 AORUS PRO` in your case
- lines 99-109 also not ... manufacturer is `Gigabyte Technology Co., Ltd.` in your case
- lines 110-124 ... this is my guess but I cannot verify it  ... can you still check if this conditions are met and maybe have a hint for me?
- lines 125-133 should not set virtual to true
- lines 134-156 should not set virtual to true (not BSD)
- lines 160-181 also here I do not think that this would match ...

Thank you for your help here. Best regards.

**Comment by Sly-Little-Fox** [comment]
Found it! It was line 167!

**Comment by Sly-Little-Fox** [comment]
It found "kvm" in this:
```
[    5.081487] kvm: Nested Virtualization enabled
```

**Comment by sebhildebrandt** [comment]
@Sly-Little-Fox Thank you! Fixed and tested! Version 5.6.20 just released.
